docs/books/admin_guide/16-about-sytemd.md

```diff
@@ -0,0 +1,416 @@
+---
+title: about systemd
+author: tianci li
+contributors:
+tags:
+  - init software
+  - systemd
+  - Upstart
+  - System V 
+---
+
+# Basic overview
+
+The **systemd**, also known as **system daemon**, is a kind of init software under GNU/linux OS. 
+
```

Comment:
While it may seem counter-intuitive, there shouldn't be a "The" at the beginning of the sentence. Simply "systemd" will suffice.